go.mod

```diff
@@ -4,119 +4,142 @@ module github.com/cosmos/ibc-go/v3
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
+replace github.com/zondax/hid => github.com/zondax/hid v0.9.0
+
 require (
 	github.com/armon/go-metrics v0.3.10
 	github.com/confio/ics23/go v0.7.0
-	github.com/cosmos/cosmos-sdk v0.45.1
+	github.com/cosmos/cosmos-proto v1.0.0-alpha7
+	github.com/cosmos/cosmos-sdk v0.46.0-alpha3.0.20220325134903-a69764f9f01b
```

Comment:
could we try this with `cosmos-sdk v0.46-beta2` and see if there are any issues? that's what's being used in gaia atm

Reply:
yes, will do next week!

Reply:
got inspired and pushed a commit!
tests pass but not sure if it's necessary to add anything to `options.ExtensionOptionChecker` or `options.TxFeeChecker`

Reply:
Thanks, @okwme! Yes, some changes will be needed there eventually when we get a `beta3` or `rc` of 0.46, but for now it should be ok.